numpy_financial/_financial.py

```diff
@@ -727,7 +727,7 @@ def rate(
     return rn
 
 
-def irr(values, *, guess=None, tol=1e-12, maxiter=100, raise_exceptions=False):
+def irr(values, raise_exceptions=False):
```

Comment:
The `raise_exceptions` argument should be a [keyword only argument](https://peps.python.org/pep-3102/)